aws/biganimal-preflight-aws

```diff
@@ -0,0 +1,483 @@
+#     * if there is enough vCPU quota left for this PostgreSQL cluster type
+#       in your AWS Account
+#     * if there is enough NLB (Network Load Balancer) quota left to expose the service
+#       for you to access the PostgreSQL cluster
+#
+# The output of this script tells any unsatisfied condition and report in the form
+# of table.
+#
+# For more details, please refer to:
+#  https://www.enterprisedb.com/docs/biganimal/latest/getting_started/01_check_resource_limits
+#
+set -e
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+[ "${BASH_VERSINFO:-0}" -lt 4 ] && echo -e "${RED}This script does not support Bash version 3 and below!${NC}" && show_help && exit 1
+CURRENT_PATH=$(pwd)
+
+function show_help()
+{
+    echo -e "Usage: ${GREEN}$0 [options] <account-id> <region>${NC}"
+    echo ""
+    echo "Arguments:"
+    echo "    <account-id>             AWS account ID for BigAnimal deployment"
+    echo "    <region>                 AWS region for BigAnimal deployment"
+    echo
+    echo "Options:"
+    echo "    -h, --help               Print this help message"
+    echo "    -i, --instance-type      AWS Instance Type for BigAnimal cluster, e.g., r5.large"
+    echo "    -a, --high-availability  Plan for BigAnimal cluster with high availability enabled"
+    echo "    -e, --endpoint           Network endpoint flavor for BigAnimal cluster"
+    echo "    -r, --activate-region    Include region activation, if no clusters exist in region"
+    echo "    --onboard                Check if the user and account are configured appropriately"
+    echo
+    echo "Behavior defaults to --onboard if no other options provided."
+    echo
+    echo "Examples:"
+    echo "    $0 --onboard 123456789012 us-east-1"
+    echo "    $0 -i r5.2xlarge --high-availability -e private -r 123456789012 us-east-1"
+    echo
+    echo "Available regions are: "
+    echo -e "  ${GREEN}${AVAILABLE_REGIONS[@]}${NC}"
+    echo "Available instance types are: "
+    echo -e "  ${GREEN}${AVAILABLE_INSTANCETYPE[@]}${NC}"
+    echo "Available endpoint flavors are: "
+    echo -e "  ${GREEN}${AVAILABLE_ENDPOINTS[@]}${NC}"
+}
+
+function suggest()
+{
+    local what=$1
+    local highlight=$2
+
+    if [ "$highlight" = 'alert' ]; then
+        echo -e "${RED}${what}${NC}"
+    else
+        echo -e "${GREEN}${what}${NC}"
+    fi
+}
+
+AVAILABLE_ENDPOINTS=(
+  public
+  private
+)
+
+AVAILABLE_REGIONS=(
+  eu-north-1
+  ap-south-1
+  eu-west-3
+  eu-west-2
+  eu-west-1
+  ap-northeast-3
+  ap-northeast-2
+  ap-northeast-1
+  sa-east-1
+  ca-central-1
+  ap-southeast-1
+  ap-southeast-2
+  eu-central-1
+  us-east-1
+  us-east-2
+  us-west-1
+  us-west-2
+)
+
+AVAILABLE_INSTANCETYPE=(
+  r5.large
+  r5.xlarge
+  r5.2xlarge
+  r5.4xlarge
+  r5.8xlarge
+  r5.12xlarge
+  r5.16xlarge
+  r5.24xlarge
+  r5b.large
+  r5b.xlarge
+  r5b.2xlarge
+  r5b.4xlarge
+  r5b.8xlarge
+  r5b.12xlarge
+  r5b.16xlarge
+  r5b.24xlarge
+  c5.large
+  c5.xlarge
+  c5.2xlarge
+  c5.4xlarge
+  c5.8xlarge
```

Comment:
we don't have `c5.8xlarge` & `c5.16xlarge`. We should replace it with `c5.9xlarge` & `c5.18xlarge` respectively.

Reply:
You're right @neel5481, I'll update that accordingly, thanks!